beetsplug/mbsync.py

```diff
@@ -109,6 +120,15 @@ def albums(self, lib, query, move, pretend, write):
 
             items = list(a.items())
 
+            valid_albumid = re.match(mb_regex, a.mb_albumid)
+
+            # Do we have a valid MusicBrainz AlbumId
+
```

Comment:
Please delete this blank line to visually associate the comment with the code it describes.

Also, did you mean to end this sentence with a question mark?

Reply:
Updated